chef/cookbooks/ceph/resources/client.rb

```diff
@@ -1,6 +1,13 @@
 actions :add
 default_action :add
 
+# Whether we have ceph cluster deployed with non standard configuration
+# and ceph.conf and admin keyring is placed somewhere else than in /etc/ceph
+# This can happened when we want to use external ceph cluster deployed 
+# without crowbar
```

Comment:
Thank you Ralf for helping with comment.
Answers on your question should be:
1. We can use multiple ceph clusters and each of them should have separete ceph.<clustername>.conf file and admin keyring. It can be putted in different locations, but they can be also created in /etc/ceph/ directory.
For example we have 3 ceph clusters installed, configuration will look like
Cluster1
/etc/ceph/ceph.cluster1.conf
/etc/ceph/ceph.cluster1.client.admin.keyring
Cluster2
/etc/ceph/ceph.cluster1.conf
/etc/ceph/ceph.cluster1.client.admin.keyring
Cluster3
/etc/ceph/ceph.cluster1.conf
/etc/ceph/ceph.cluster1.client.admin.keyring
2. Default location for ceph is
/etc/ceph/ceph.conf
/etc/ceph/ceph.client.admin.keyring
it is when there is used only one ceph cluster.